src/status_im/multiaccounts/core.cljs

```diff
@@ -43,10 +43,11 @@
   [contact profile contact-identity]
   (let [me? (= (:public-key profile) contact-identity)]
     (if me?
-      [(or (:preferred-name profile)
-           (:display-name profile)
-           (:primary-name contact)
-           (gfycat/generate-gfy contact-identity))]
+      [(or
+        (when (seq (:preferred-name profile)) (:preferred-name profile))
+        (when (seq (:display-name profile)) (:display-name profile))
+        (when (seq (:primary-name contact)) (:primary-name contact))
```

Comment:
I think it's better to use string/blank? for strings to check for emptiness.
It's more performant.

Reply:
Thanks @ibrkhalil , resolved.